### PR TITLE
Update full node to 2.1.0

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: src/utils/consts.ts [hunk 1]
```diff
@@ -9,27 +8,27 @@ export interface NetworkConfig {
...
-  }
-  return {
-    networkId: network === 'testnet' ? 1 : 4,
-    groupIndex: groupOfAddress(deployment.deployerAddress),
-    factoryId: deployment.contracts.TokenPairFactory.contractId,
-    routerId: deployment.contracts.Router.contractId
+  try {
+    const deployments = loadDeployments(networkId)
+    return {
+      networkId: networkId === 'testnet' ? 1 : 4,
```
Comment (polarker) [inline src/utils/consts.ts:25]: Is this number id still needed?

Comment (Lbqds) [reply src/utils/consts.ts]: No need, deleted

**Comment by nop33** [comment]
Perfect, thanks for the feedback! Closing PR.

**Comment by nop33** [comment]
Oops, sorry, wrong PR 😅 I wanted to close this one: #12

**Pull request merged**